presto-raptor/src/main/java/com/facebook/presto/raptor/systemtables/ShardMetadataRecordCursor.java

```diff
@@ -65,6 +66,10 @@
     private static final String TABLE_NAME = "table_name";
     private static final String MIN_TIMESTAMP = "min_timestamp";
     private static final String MAX_TIMESTAMP = "max_timestamp";
+    private static final String MIN_DATE = "min_date";
+    private static final String MAX_DATE = "max_date";
+
+    private static final String DEFAULT_COLUMN_NAME = "null";
```

Comment:
It's not a column name. Instead, it seems to be a placeholder for an unused column. Choose a better name for it.